test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,174 @@
+
+   ```bash 
+   kubectl config use-context wds1
+   ```
+
+   Output:
+   ```console 
+   Switched to context "wds1".
+   ```
+   
+   Optionally, check the kubeconfig contexts: 
+   ```bash 
+   kubectl config get-contexts
+   ```
+   
+   Output:
+   ```console 
+   CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+             cluster1        kind-cluster1   kind-cluster1   
+             cluster2        kind-cluster2   kind-cluster2   
+             its1            its1-cluster    its1-admin      
+             kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *         wds1            wds1-cluster    wds1-admin      default
+             wds2            wds2-cluster    wds2-admin      default
+   ```
+
+   Second, create the binding Policy and namespace for your experiment using the following commands:
+
+   ```bash 
+   cd $CL2_DIR/testing/load/
+   kubectl apply -f long-duration-exp-setup.yaml
+   ```
+
+   Lastly, in the following set ``--kubeconfig`` flag to the path of the kubeconfig file used above and use the newly added Kubestellar provider (`--provider=ks`) and run the following command to create the workload:
+
+   ```bash
+   cd $CL2_DIR
+   go run cmd/clusterloader.go --testconfig=./testing/load/long-duration-exp-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
+   ```
+    
+   The generated workload is a pod that sleeps for 20 seconds([see here](workloads/long-duration-exp-workload.yaml)) and created using the clusterloader `SteppedLoad` tuningSet (burstSize=1 and stepDelay=60 sec). SteppedLoad defines a load that generates a burst of a given size every X seconds, see more [here](https://github.com/kubernetes/perf-tests/blob/6ec633314941e74d8d8e31c83ac8711e7c3d9832/clusterloader2/docs/design.md#tuning-set).
+
+
+5. Run the metrics collection script: collect the creation and status_update timestamps for the benchmark workload objects proceed as following
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash 
+   cd .$HOME/kubestellar/test/performance/common
+   ```
+
+   b) Run the metrics collection script:
+
+   ```bash
+   python3 metrics_collector.py <kubeconfig> <wds-context-name> <its-context-name> <wec-context-name> <number-of-namespaces> <output-directory> <exp-type> <number-of-pods> <watch-interval>
+   ```
+
+   For example:
+   
+   ```bash 
+   python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 1 $HOME/data l 10 30
+   ```
+
+   Observations: 
+      - The execution of the above python script requires the installation of the dependencies listed in `requirements.txt`.
+      - The script creates a custom controller that deletes a pod after reaching the completed state 
+      - The workload generation tool (i.e., clusterloader2) launched at `step 4` will create another pod after 1 minute with a different name.
```

Comment:
Putting code quotes around "step 4" makes it look odd. Here "step 4" is not something in code, it is a reference to a point in this document. I think it will look better with no quoting, "step 4" in plain English means what you are trying to say here.

Reply:
Fixed! Removed the quotes.